Sources/Money/Money.swift

```diff
@@ -72,7 +72,7 @@ extension Money {
 }
 
 extension Money {
-    /// Subtracts one monetary amount from another.
+    /// Flips the sign (plus to minus or vice versa) of the monetary amount.
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/Negative_number):

> The negation of a negative number is the corresponding positive number.

So how about this?

```suggestion
    /// Negates the monetary amount.
```